src/components/TokenSelect/CommonBases.tsx

```diff
@@ -0,0 +1,66 @@
+import { Currency } from '@uniswap/sdk-core'
+import Row from 'components/Row'
+import TokenImg from 'components/TokenImg'
+import { BASES_TO_CHECK_TRADES_AGAINST } from 'constants/routing'
+import { transparentize } from 'polished'
+import styled from 'styled-components/macro'
+import { ThemedText } from 'theme'
+import { currencyId } from 'utils/currencyId'
+
+const BasesContainer = styled(Row)`
+  margin: 0 1.25em;
+`
+
+const BaseWrapper = styled.div<{ active?: boolean }>`
+  align-items: center;
+  background-color: ${({ theme, active }) => (active ? transparentize(0.76, theme.active) : 'transparent')};
```

Comment:
Would it be more readable if you took the ternary properties, which match `:hover`, and put them in a separate css macro, so that you don't need to write it out twice? Then you can just use `${({ active }) => active && activeCss};`.